vue-frontend/src/components/jobs/LifeAtCanopas.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div>
+    <div class="canopas-life-div normal-text">
+      <div class="canopas-life-title">
+        <div class="header-text canopas-gradient-text text-center">
+          Life at Canopas
+        </div>
+      </div>
+      <div class="description mt-5 mb-5">
+        <div class="text-center">
+          We wanted to create an environment that helps you get out of bed in
+          the morning and an office that creates excellent work and encourages
+          fun. As we play the infinite game, it is not about winning or losing,
+          it's about striving to be better.
+        </div>
+      </div>
+    </div>
+    <swiper
+      :pagination="pagination"
+      :slidesPerView="1"
+      :centeredSlides="true"
+      :loop="true"
+      :spaceBetween="20"
+      :modules="modules"
+      class="swiper"
+    >
+      <swiper-slide
+        v-for="(sider, index) in slides"
+        :key="index"
+        class="swiper-wrapper"
+      >
+        <img :src="sider.image" class="swiper-slide" />
+      </swiper-slide>
+    </swiper>
+    <div class="swiper-pagination mb-5"></div>
+  </div>
+</template>
+
+<script>
+// Import Swiper Vue.js components
+import { Swiper } from "../../../node_modules/swiper/vue/swiper";
+import { SwiperSlide } from "../../../node_modules/swiper/vue/swiper-slide";
+import SwiperCore, { Pagination } from "swiper";
+// Import Swiper styles
+import "../../../node_modules/swiper/swiper.min.css";
+import "../../../node_modules/swiper/modules/pagination/pagination.min.css";
+SwiperCore.use([Pagination]);
+
+export default {
+  data() {
+    return {
+      slides: [
+        { id: 1, image: require("@/assets/images/canopas-life/1.webp") },
```

Comment:
Let's use svg instead of webp , also rename directory from `canopas-life` to `life` and images to `jobs_canopas_life_1.svg`